### DISCUSSION TITLE
V21.4.4-v1

### DISCUSSION THREAD

**Discussion by austinsonger** [Secure-Compliance-Solutions-LLC/GVM-Docker#307] [New Release]
## What's Changed
* Final Fix to build Image by @Dexus in https://github.com/Secure-Compliance-Solutions-LLC/GVM-Docker/pull/305
* v210404.0.1 by @Dexus in https://github.com/Secure-Compliance-Solutions-LLC/GVM-Docker/pull/306

**Full Changelog**: https://github.com/Secure-Compliance-Solutions-LLC/GVM-Docker/compare/v21.4.4...V21.4.4-v1

<hr /><em>This discussion was created from the release <a href='https://github.com/Secure-Compliance-Solutions-LLC/GVM-Docker/releases/tag/V21.4.4-v1'>V21.4.4-v1</a>.</em>

**Comment by netbix** [comment]
I tried the new build with debian and the new install works, the problem is when trying to upgrade above the alpine version. the database in alpine is 13 in debian 11 and the old database doesn't work
